microsoft/ts-fix
# Suggest autofixes for declaration diagnostics

This is to allow autofixes for errors like https://github.com/microsoft/TypeScript/pull/58260.

Test plan:

- Build and install the TS branch above
- Have a TS project with isolatedDeclarations enabled and containing exports whose types cannot be inferred trivially.
- Run `ts-fix -t <path_to_tsconfig> -e9013 -f add-annotation --write --ignoreGitStatus` and verify that type annotations are automatically added where necessary.

Fixes #32
